.eslintrc.cjs

```diff
@@ -19,6 +19,8 @@ const config = {
 		'import/parsers': {
 			[require.resolve('@typescript-eslint/parser')]: ['.ts', '.tsx'],
 		},
+		'import-sorting/known-framework': '^react(/|-dom|-router|$)',
+		'import-sorting/known-first-party': '^~',
```

Comment:
I don't think we need these settings here, since this project won't have those kinds of imports 🙂